includes/admin/views/html-admin-page-addons.php

```diff
@@ -25,19 +25,45 @@
 
 	<?php if ( $sections ) : ?>
 		<ul class="subsubsub">
-			<?php foreach ( $sections as $section_id => $section ) : ?>
-				<li><a class="<?php echo $current_section === $section_id ? 'current' : ''; ?>" href="<?php echo admin_url( 'admin.php?page=wc-addons&section=' . esc_attr( $section_id ) ); ?>"><?php echo esc_html( $section->title ); ?></a><?php echo ( end( $section_keys ) !== $section_id ) ? ' |' : ''; ?></li>
+			<?php foreach ( $sections as $section ) : ?>
+				<li>
+					<a
+						class="<?php echo $current_section === $section->slug ? 'current' : ''; ?>"
+						href="<?php echo admin_url( 'admin.php?page=wc-addons&section=' . esc_attr( $section->slug ) ); ?>">
+						<?php echo esc_html( $section->label ); ?>
+					</a>
+				</li>
 			<?php endforeach; ?>
 		</ul>
-		<br class="clear" />
-		<?php if ( 'featured' === $current_section ) : ?>
+
+		<?php if ( isset( $_GET['search'] ) ) : ?>
+			<h1 class="search-form-title" >
+				<?php _e( 'Showing Search results for:', 'woocommerce' ); ?>
+				<strong><?php echo esc_html( $_GET['search'] ); ?></strong>
```

Comment:
The search term should really be a placeholder inside the translated string, which will allow RTL languages to move things around:

```
printf( __( 'Showing search results for: %s', 'woocommerce' ), '<strong>' . esc_html( $_GET['search'] ) . '</strong>' ) );
```